acceptance/lib/puppet/acceptance/common_utils.rb

```diff
@@ -30,6 +30,16 @@ def user_absent(host, username)
       end
     end
 
+    module PackageUtils
```

Comment:
- did not promote to 60.x - CAUtils, CronUtils
    - added PackageUtils @loopinu